models/issue.go

```diff
@@ -715,8 +715,8 @@ func updateIssueCols(e Engine, issue *Issue, cols ...string) error {
 }
 
 // UpdateIssueCols only updates values of specific columns for given issue.
-func UpdateIssueCols(issue *Issue, cols ...string) error {
-	return updateIssueCols(x, issue, cols...)
+func UpdateIssueCols(ctx DBContext, issue *Issue, cols ...string) error {
```

Comment:
Not for this PR, but as a side note: calling this from outside `models` exposes too much of the internal structure IMHO. It doesn't seem right for other modules to give a list of table columns.